packages/toolpad-app/src/toolpad/propertyControls/GridColumns.tsx

```diff
@@ -64,6 +76,15 @@ function GridColumnsPropEditor({
   const { bindings } = usePageEditorState();
   const [editColumnsDialogOpen, setEditColumnsDialogOpen] = React.useState(false);
   const [editedIndex, setEditedIndex] = React.useState<number | null>(null);
+  const { dom } = useDom();
+  const toolpadComponents = useToolpadComponents(dom);
```

Comment:
Would a `useCustomComponents` hook possibly be useful in the future? Just thinking it could make things a bit more readable and reusable, but if we don't have widespread use it for it yet we can also just start like this!

Reply:
I'd prefer to wait if we see a need, it would make it easier to know if the used abstraction makes sense 🤔